draft-gbb-ccamp-optical-path-computation-yang.txt

```diff
@@ -1669,22 +1669,51 @@ Internet-Draft      Yang for Optical Path Computation     September 2022
 
              Figure 5: Flexi-grid path computation YANG module
 
-5.  Manageability Considerations
 
-   TBD.
 
 
 
 
 
-Busi, et al.              Expires 12 March 2023                [Page 30]
+
+
+Busi, et al.              Expires 16 March 2023                [Page 30]
 
 Internet-Draft      Yang for Optical Path Computation     September 2022
 
 
+5.  Manageability Considerations
+
+   This document provides a method for requesting path computations for
+   WSON and Flexi-Grid tunnels.  Consideration of mechanisms to gather
+   and collate information required for the path computations will be
+   necessary.  Furthermore, storing path computation requests and
+   responses and triggering actions according to configured will also
+   need to be carefully managed and secured.
+
```

Comment:
....according to configured will also need...
I would think the words missing would be "tunnels" probably but the sentence has to be completed